examples/hybrid/driver.jl

```diff
@@ -165,8 +159,7 @@ function additional_tendency!(Yₜ, Y, p, t)
         CA.precipitation_tendency!(Yₜ, Y, p, t, colidx, p.precip_model)
     end
     # TODO: make bycolumn-able
-    (; non_orographic_gravity_wave) = p.tendency_knobs
-    non_orographic_gravity_wave && CA.gravity_wave_tendency!(Yₜ, Y, p, t)
+    CA.gravity_wave_tendency!(Yₜ, Y, p, t, p.atmos.gravity_wave)
```

Comment:
Should we change the `gravity_wave_tendency!` function to dispatch on `atmos.gravity_wave` then?

Reply:
Good catch! I've updated that